branchinfo.txt

```diff
@@ -1,9 +1,15 @@
 # This is a file containing environment variables specific to this branch
 # Any line that is not blank and does not start with '#' is interpreted as a variable to set
 # Each line is expected to be in the format "[Name]=[Value]".
+#
+#
+# !!! WARNING !!! WARNING !!! WARNING !!! WARNING !!! WARNING !!! WARNING
+#       "master" branch temporarily produced 1.0.1 builds. The value here
```

Comment:
When we service netcore.app I would imagine you'd service from the rel branch rev-ing patch versions.

When you ship master I'd expect it would be 1.1.0 and not a servicing train. Cc @weshaggard.